torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -627,7 +607,10 @@ class TensorExprFuser {
   // until there is nothing we can pull in.
   std::pair<graph_node_list::iterator, bool> createFusionGroup(
       Node* fusion_node) {
-    if (min_group_size_ == 1) {
```

Comment:
Unrelated to this PR, but maybe we should enable min_group_size == 1 by default? IIRC, @Chillee experiments showed that the perf was reasonable

Reply:
Not sure about whether we should do it now, but definitely something to reconsider when we support dynamic shapes